### PR TITLE
start new editors post

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

File: content/blog/2021-10-12-editors2021/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: 'Announcing New Software Peer Review Editors: Emily Riederer, Adam Sparks, and ... Bueller?'
+author: Stefanie Butland
+date: '2021-10-27'
```
Comment (stefaniebutland) [inline content/blog/2021-10-12-editors2021/index.md:4]:
```suggestion
date: '2021-10-12'
```

File: content/blog/2021-10-12-editors2021/index.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+{{< /imgtxt >}}
+
+Emily on [GitHub](https://github.com/emilyriederer), [Twitter](https://twitter.com/emilyriederer), [Website](https://emilyriederer.netlify.com/)
+
+> I had the great fortune of stumbling upon rOpenSci relatively earliy in my career as an R programmer, and it's not exaggeration to say that this organization has had one of the single largest impacts on my career. rOpenSci's meticulous documentation helped me to embrace best packages in my development work. Perhaps even more impactful is how watching rOpenSci work shaped my perspective on community building, leadership, and teamwork. I have always admired the well-oiled machine that rOpenSci has created -- sustaining massive value with limited individual overhead by clearly defining expectations, providing opinionated infrastrcutre, and cultivating a phenomenal communtiy. I'm thrilled to be taking a formal role in an organization that has had such an impact on me.
```
Comment (stefaniebutland) [inline content/blog/2021-10-12-editors2021/index.md:36]:
```suggestion
> I had the great fortune of stumbling upon rOpenSci relatively early in my career as an R programmer, and it's not an exaggeration to say that this organization has had one of the single largest impacts on my career. rOpenSci's meticulous documentation helped me to embrace best practices in my development work. Perhaps even more impactful is how watching rOpenSci work shaped my perspective on community building, leadership, and teamwork. I have always admired the well-oiled machine that rOpenSci has created -- sustaining massive value with limited individual overhead by clearly defining expectations, providing opinionated infrastructure, and cultivating a phenomenal community. I'm thrilled to be taking a formal role in an organization that has had such an impact on me. 
```

Comment (stefaniebutland) [reply content/blog/2021-10-12-editors2021/index.md:36]: @emilyriederer I changed "embrace best packages" to "embrace best practices"
Is that what you meant?

Thanks so much for the generous quote.

File: content/blog/2021-10-12-editors2021/index.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+Editors [manage the review process](https://devguide.ropensci.org/editorguide.html), performing initial package checks, identifying reviewers, and moderating the process until the package is accepted by reviewers and transferred to rOpenSci. 
+To address the scope and volume of packages submitted for review, it's critical that we have a team of editors with complementary expertise.
+Adam brings his broad experience with API packages and his experience from agricultural research in academia and government.
+Emily is known for openly sharing her processes-oriented approach to development based in her experience in industry.
+
+Meet our new editors!
+
+## Emily Riederer
+
+{{< imgtxt src = "emily-riederer.jpg" width = "200" alt = "headshot of Emily Riederer">}}
```
Comment (stefaniebutland) [inline content/blog/2021-10-12-editors2021/index.md:26]:
```suggestion
{{< imgtxt src = "/img/team/emily-riederer.jpg" width = "200" alt = "headshot of Emily Riederer">}} 
```

**Review by stefaniebutland — COMMENTED** [review]
Fixed link to Emily's picture. 
I'll need (help) to update this branch since Adam's pic was added after the branch was created.

File: content/blog/2021-10-12-editors2021/index.md (hunk 2 above)
Comment (emilyriederer) [reply content/blog/2021-10-12-editors2021/index.md:36]: Oh yikes - thank you, @stefaniebutland ! That is definitely what I meant. I appreciate the typo fixes also. I wish my editor announcement hadn't needed so much editing 🙂

**Pull request merged**